README.md

```diff
@@ -79,29 +65,38 @@ Click the **Deploy Azure** button to  begin.
 
 **NOTE:** In general it takes over an hour for the deployment to complete.
 
-<a target="_blank" href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fteradici%2Fdeploy%2Fmaster%2Fazuredeploy.json">
+<a target="_blank" href="https://portal.azure.com/">
     <img src="http://azuredeploy.net/deploybutton.png"/>
 </a>
 
 1. Select the Microsoft Azure account you want to access.
 1. Enter your Password and click **Sign in.**
-1. On the Customized Template page create a new Resource group by selecting the **Create New** icon and entering a name for the group. The Resource Group should be empty when you access the page.
-    * You can also select a pre-defined Resource group by selecting the **Use Existing** icon and clicking on one of the groups from the dropdown menu.
-4. Select a location from the dropdown menu.
-5. Enter a Username for the **domainAdminUsername**. This is a new account.
-6. Enter a password for the **domainAdminPassword**. This is a new password.
-7. Enter a **domainName** and ensure it finishes in **.com**.
-8. Enter your **AzureAdminUsername**. This must be the same account you logged into from step 1.
-9. Enter your **AzureAdminPassword**. This must be the same password you used to log in from step 2.
-10. Enter the CAS license registration code for the **registrationCode**.
-11. Use the default addresses that are pre-entered for the **adminVMBlobSource** and **_artifactsLocation**. 
-12. Read the Terms and Conditions and once you are satisified with the information you have entered click the **I Agree** icon.
-13. Click **Purchase** to begin deployment.
+1. Launch an Azure Cloud Shell Powershell instance. See [Overiew of Azure Cloud Shell](https://docs.microsoft.com/en-ca/azure/cloud-shell/overview?view=azurermps-5.0.0) for details on how to set this up. The rest of the setup steps will be run in the Azure Cloud Shell. It is recomended to maximize the Cloud Shell window so prompts can be properly seen.
+1. Change to your Cloud Drive directory. 
+    ```
+    cd $HOME\CloudDrive\
```

Comment:
I've had that issue as well. It makes sense to do it to $HOME instead to get around this. I initially chose CloudDrive since it's persistent storage, but from a customer experience perspective it might make more sense to not use it. We could add a test path test first, but that would probably double the number of lines of codes the user would need to run so I'm a bit on the fence for adding something that isn't completely necessary.

Reply:
Yeah just going to $HOME makes sense to me. (I emailed that and somehow it got lost... Anyway - learning the tool :) )